test/external_project/install-sharg.cmake

```diff
@@ -4,6 +4,8 @@
 
 cmake_minimum_required (VERSION 3.14)
 
+cmake_policy (SET CMP0135 NEW)
```

Comment:
Can you please add a comment what this policy is for.
Maybe that explains why it is set already. Otherwise, please enlighten me :)

Reply:
I'll add a link to https://cmake.org/cmake/help/latest/policy/CMP0135.html